opts.go

```diff
@@ -271,3 +314,21 @@ func loadFromConfDir(c *libcni, max int) error {
 	c.networks = append(c.networks, networks...)
 	return nil
 }
+
+func checkPluginExists(c *libcni, confList *cnilibrary.NetworkConfigList) error {
+	for _, plug := range confList.Plugins {
+		plugin := plug.Network.Type
+
+		if !fileExistsInDir(c.pluginConfDir, plugin) {
```

Comment:
@mikebrow I think I have spotted a bug. If the user has multiple CNI plugin binaries and the specified cni plugin is in the last checked directory then this will false.